### PR TITLE
tests/aspects: Replace \ with () for line continuation

### LINKED ISSUE #3928: tests/aspects: Replace \ with () for line continuation
```
tests/bearlib/aspects/InstanceTest.py:            assert str(exc.value) \
tests/bearlib/aspects/InstanceTest.py:            assert str(exc.value) \
tests/bearlib/aspects/SubTest.py:                    assert getattr(aspect, name) == taste_values[name] \
tests/bearlib/aspects/SubTest.py:        assert SubAspect('py', **SubAspect_taste_values) \
tests/bearlib/aspects/SubTest.py:        assert not SubAspect('py') \
tests/bearlib/aspects/SubTest.py:        assert not SubAspect('py', **SubAspect_taste_values) \
tests/bearlib/aspects/SubTest.py:        assert not SubAspect('py', **SubAspect_taste_values) \
tests/bearlib/aspects/TasteTest.py:            assert getattr(using_custom_values, name) \
```
Use of `\` for line continuation is not allowed in coala. These are the instances where `\` is used instead of enclosing braces for line continuation.

### PR DESCRIPTION
**Pull request by sinwar**
Fixes #3928

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by twinone** [comment]
ack c1be5d7bc43f2608cb291f150b8b9692cdaa3710

File: tests/bearlib/aspects/SubTest.py [hunk 1]
```diff
@@ -31,28 +31,28 @@ def test_tastes(self, SubAspect, SubAspect_tastes):
             taste_values = aspect.tastes
             for name, taste in SubAspect_tastes.items():
                 if not taste.languages or language in taste.languages:
-                    assert getattr(aspect, name) == taste_values[name] \
-                        == taste.default
+                    assert getattr(aspect, name) == (
```
Comment (Udayan12167) [inline tests/bearlib/aspects/SubTest.py]: `(getattr(aspect, name) == (taste_values[name]) == taste.default)` and then break the line.

File: tests/bearlib/aspects/SubTest.py [hunk 2]
```diff
@@ -31,28 +31,28 @@ def test_tastes(self, SubAspect, SubAspect_tastes):
...
-            == SubAspect('py', **SubAspect_taste_values)
+        assert SubAspect('py', **SubAspect_taste_values) == (
+            SubAspect('py', **SubAspect_taste_values))
         assert not SubAspect('py') == RootAspect('py')
         assert not SubAspect('py') == Root('py')
-        assert not SubAspect('py') \
-            == SubAspect('py', **SubAspect_taste_values)
-        assert not SubAspect('py', **SubAspect_taste_values) \
-            == SubAspect('py')
+        assert not SubAspect('py') == (
```
Comment (Udayan12167) [inline tests/bearlib/aspects/SubTest.py]: bracket after `not`

File: tests/bearlib/aspects/SubTest.py [hunk 3]
```diff
@@ -31,28 +31,28 @@ def test_tastes(self, SubAspect, SubAspect_tastes):
...
+            SubAspect('py', **SubAspect_taste_values))
+        assert not SubAspect('py', **SubAspect_taste_values) == (
+            SubAspect('py'))
 
     def test__ne__(self, RootAspect, SubAspect, SubAspect_taste_values):
         assert not SubAspect('py') != SubAspect('py')
-        assert not SubAspect('py', **SubAspect_taste_values) \
-            != SubAspect('py', **SubAspect_taste_values)
+        assert not SubAspect('py', **SubAspect_taste_values) != (
```
Comment (Udayan12167) [inline tests/bearlib/aspects/SubTest.py]: same here

File: tests/bearlib/aspects/TasteTest.py [hunk 4]
```diff
@@ -26,5 +26,5 @@ def test__get__(
         for name, taste in SubAspect_tastes.items():
             assert getattr(SubAspect, name) is taste
             assert getattr(using_default_values, name) == taste.default
-            assert getattr(using_custom_values, name) \
-                == SubAspect_taste_values[name]
+            assert getattr(using_custom_values, name) == (
```
Comment (Udayan12167) [inline tests/bearlib/aspects/TasteTest.py]: same here

**Comment by Udayan12167** [comment]
unack c1be5d7

**Comment by userzimmermann** [comment]
@sinwar @twinone @Udayan12167 Why are people so embarrassed by my cute backslashes? ;D

File: tests/bearlib/aspects/SubTest.py (hunk 1 above)
Comment (userzimmermann) [inline tests/bearlib/aspects/SubTest.py]: @sinwar 

```python
assert (getattr(aspect, name) == taste_values[name] ==
        taste.default) 
```

:)

**Comment by meetmangukiya** [comment]
@sinwar does this mean you want to unassign yourself as well

**Pull request closed without merging**